src/core/BundleSplit.ts

```diff
@@ -42,6 +42,7 @@ export class BundleSplit {
         config.plugins = [].concat(config.plugins || [])
         config.standalone = false;
         const fuse = FuseBox.init(config);
+        new Bundle(name, fuse, producer);
```

Comment:
Well, there needs to be a different solution. As your approach might solve your own issue, but will break everything else.
https://github.com/fuse-box/fuse-box-examples/tree/master/examples/react-code-splitting